pandas-stubs/core/arrays/datetimelike.pyi

```diff
@@ -35,13 +35,22 @@ class TimelikeOps:
     def unit(self) -> TimeUnit: ...
     def as_unit(self, unit: TimeUnit) -> Self: ...
     def round(
-        self, freq, ambiguous: TimeAmbiguous = ..., nonexistent: TimeNonexistent = ...
+        self,
```

Comment:
Any reason why your PR would change the formatting? I don't see a change in the argument.

Reply:
As long as the pre-commit passes that's not a big deal, I was trying to find a change and kept comparing the line but could not find any!